common.mk

```diff
@@ -146,29 +146,29 @@ FIRTOOL_TARGETS = \
 	$(FIRTOOL_FILELIST) \
 	$(FIRTOOL_BB_MODS_FILELIST)
 
-ifeq (,$(ENABLE_CUSTOM_FIRRTL_PASS))
-	REPL_SEQ_MEM = none
-	TOP_TARGETS = none
-	HARNESS_TARGETS = none
-	TRANSFORMS = barstools.tapeout.transforms.GenerateTop
-else
-	REPL_SEQ_MEM = --infer-rw --repl-seq-mem -c:$(MODEL):-o:$(TOP_SMEMS_CONF)
-	TOP_TARGETS = $(TOP_SMEMS_CONF)
-	HARNESS_CONF_FLAGS = -thconf $(HARNESS_SMEMS_CONF)
-	HARNESS_TARGETS = $(HARNESS_SMEMS_CONF)
-	TRANSFORMS = barstools.tapeout.transforms.GenerateTopAndHarness
-endif
+SFC_REPL_SEQ_MEM = --infer-rw --repl-seq-mem -c:$(MODEL):-o:$(SFC_SMEMS_CONF)
 
 
 # DOC include start: FirrtlCompiler
-# hack: lower to middle firrtl if Fixed types are found
-$(HARNESS_TARGETS) $(TOP_TARGETS) $(FIRTOOL_TARGETS) &: $(FIRRTL_FILE) $(FINAL_ANNO_FILE) $(VLOG_SOURCES)
+# This step can take either one of two paths. The first path is when SFC
+# compiles Chisel to CHIRRTL, and FIRTOOL compiles CHIRRTL to Verilog. Otherwise,
+# when custom FIRRTL transforms are included or if a Fixed type is used within
+# the dut, SFC compiles Chisel to LowFIRRTL and FIRTOOL compiles it to Verilog.
+# Users can indicate to the Makefile of custom FIRRTL transforms by setting the
+# "ENABLE_CUSTOM_FIRRTL_PASS" env variable.
```

Comment:
```suggestion
# "ENABLE_CUSTOM_FIRRTL_PASS" variable.
```

Since this can be set by the Makefile, as a Make arg, etc (not just an env. var)